helm_deploy/values-prod.yaml

```diff
@@ -0,0 +1,29 @@
+---
+# Per environment values which override defaults in approved-premises-ui/values.yaml
+
+generic-service:
+  ingress:
+    hosts:
+      - short-term-accommodation-cas-2.hmpps.service.justice.gov.uk
+    contextColour: green
+    tlsSecretName: hmpps-community-accommodation-tier-2-prod-cert
+
+  env:
+    ENVIRONMENT: prod
+    APPROVED_PREMISES_API_URL: 'https://approved-premises-api.hmpps.service.justice.gov.uk'
+    INGRESS_URL: 'https://short-term-accommodation-cas-2.hmpps.service.justice.gov.uk'
+    HMPPS_AUTH_URL: 'https://sign-in.hmpps.service.justice.gov.uk/auth'
+    TOKEN_VERIFICATION_API_URL: 'https://token-verification-api.prison.service.justice.gov.uk'
+    COMMUNITY_ACCOMMODATION_API_TIMEOUT_RESPONSE: 30000
+    COMMUNITY_ACCOMMODATION_API_TIMEOUT_DEADLINE: 30000
+    IN_MAINTENANCE_MODE: true
+
+  namespace_secrets:
+    sqs-hmpps-audit-secret:
+      AUDIT_SQS_QUEUE_URL: 'sqs_queue_url'
+      AUDIT_SQS_QUEUE_NAME: 'sqs_queue_name'
+
+  allowlist: null
+
+generic-prometheus-alerts:
+  alertSeverity: digital-prison-service-dev
```

Comment:
I don't know if we use these prometheus alerts, but should this be `digital-prison-service` ?

Reply:
Good shout, this was actually removed for prod by cas3 after a convo with the platform team. I've removed it and edited the commit to include the relevant links.